File: src/main/java/com/example/hbs/psi/HbPsiElement.java
package com.example.hbs.psi;

import com.intellij.openapi.util.NlsSafe;
import com.intellij.psi.PsiElement;

/**
 * Base for all Handlebars/Mustache elements
 */
public interface HbPsiElement extends PsiElement {

  @NlsSafe
  String getName();
}

File: src/main/kotlin/com/example/hbs/psi/HbPsiElement.kt
package com.example.hbs.psi

import com.intellij.openapi.util.NlsSafe
import com.intellij.psi.PsiElement

/**
 * Base for all Handlebars/Mustache elements
 */
interface HbPsiElement : PsiElement {

    @get:NlsSafe
    val name: String
}
